### ISSUE TITLE
Why my test results in mot17 are much better than given

### ISSUE THREAD

**Issue by pjy125175** [ifzhang/FairMOT#516]
I only use mot17's training set and only test on mot17's test set.
Use the sh experiments/mot17_dla34.sh command to train.
And I got a result like this：
![2022-07-13 10-27-51屏幕截图](https://user-images.githubusercontent.com/53651859/178637950-ced8464b-470c-4526-8f59-4fa82590d7d2.png)
